gef.py

```diff
@@ -6382,21 +6412,32 @@ def __init__(self):
     @only_if_gdb_running
     def do_invoke(self, argv):
         location = None
+        show_opcodes = False
 
         kwargs = {}
         for arg in argv:
             if "=" in arg:
                 key, value = arg.split("=", 1)
                 kwargs[key] = value
 
+            elif "opcodes".startswith(arg.lower()):
+                show_opcodes = True
+
             elif location is None:
                 location = parse_address(arg)
 
         location = location or current_arch.pc
         length = int(kwargs.get("length", get_gef_setting("context.nb_lines_code")))
 
+        insns = []
+        opcodes_len = 0
         for insn in capstone_disassemble(location, length, skip=length*self.repeat_count, **kwargs):
-            text_insn = str(insn)
+            insns.append(insn)
+            opcodes_len = max(opcodes_len, len(insn.opcodes))
```

Comment:
Why is opcodes_len set to the length of the longest instruction?

Reply:
Ah, it's to make sure that the mnemonic is alined, right?

Reply:
Yup